src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,32 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
+ */
+function wp_print_template_loader_script() {
+	$wp_scripts = wp_scripts();
+
+	if ( $wp_scripts->has_delayed_inline_script() ) {
+		$output    = <<<JS
+function wpLoadAfterScripts( handle ) {
```

Comment:
This function is using ES6 syntax which would break in IE. I know that IE is [not supported](https://wordpress.org/news/2021/05/dropping-support-for-internet-explorer-11/) anymore, but given that this code will appear on the _frontend_ it seems to me sensible that it could be written in ES5 so as to not intentionally break IE when we don't have to. I don't feel strongly about this.